RHEL/7/input/oval/accounts_passwords_pam_faillock_deny_root.xml

```diff
@@ -0,0 +1,94 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwords_pam_faillock_deny_root" version="4">
+    <metadata>
+      <title>Lock out the root account after failed login attempts</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>The root account should be configured to deny access after the number of defined
+      failed attempts has been reached.</description>
+      <reference source="galford" ref_id="RHEL7_20150122" ref_url="test_attestation" />
+    </metadata>
+    <criteria>
+      <criterion test_ref="test_accounts_passwords_pam_faillock_preauth_silent_system-auth"
```

Comment:
There's a problem with this OVAL check (not that it would be syntactically incorrect or not working), but the issue is it uses duplicate (the same) ```test_ref``` elements IDS like different OVAL check:

```
]$ fgrep -rl 'test_accounts_passwords_pam_faillock_preauth_silent_password-auth' .
./shared/oval/accounts_passwords_pam_faillock_deny.xml
./RHEL/7/input/oval/accounts_passwords_pam_faillock_deny_root.xml
```
What this means being there are currently two OVAL checks having the same ```test_ref``` IDS, but referencing different actual OVAL ```<test>``` element implementation. Since currently SSG allows to build such content (this is a bug tracked now under: https://github.com/OpenSCAP/scap-security-guide/issues/1275), this leads to ```ssg-rhel7-oval.xml``` file referencing just the first OVAL ```test_ref``` implementation.

Using such OVAL file on correctly configured system, and scanning the ```accounts_passwords_pam_faillock_deny_root``` leads to ```error``` state:
```
# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.2 (Maipo)
# oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_common_tailored --tailoring-file /tmp/tailoring.xml /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml 
This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping http://www.redhat.com/security/data/oval/Red_Hat_Enterprise_Linux_7.xml file which is referenced from XCCDF content
Title   Configure the root Account for Failed Password Attempts
Rule    xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny_root
Result  error
```

This is because the generated ```ssg-rhel7-oval.xml``` file instead of the test implementation for the second rule, contains test implementation for the first rule (since ```test_ref``` elements) are the same.

To fix this we need to:
* ```test_ref``` IDS in newly added ```scap-security-guide/RHEL/7/input/oval/accounts_passwords_pam_faillock_deny_root.xml``` to have unique IDs (different from those in ```./shared/oval/accounts_passwords_pam_faillock_deny.xml``` OVAL check implementation (this can be done within this pull request),
* and fix the SSG build system not to allow to build the OVAL in cases like this (but rather exit with error), since it might lead to hard to debug issues (this is tracked under https://github.com/OpenSCAP/scap-security-guide/issues/1275 )

@redhatrises 
Please change the ```test_ref``` IDs for this OVAL check they not to collide with those from shared pam_faillock_deny check. Then I will re-review.